app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -3,6 +3,8 @@ class DradisDatatable {
     this.$table = $(tableElement);
     this.dataTable = null;
     this.tableHeaders = Array.from(this.$table[0].querySelectorAll('thead th'));
+    this.$paths = this.$table.closest('[data-behavior~=datatable-paths]');
+    this.itemName = this.$table.data('itemName');
```

Comment:
TIL that this works, but let's use this for consistency:
```suggestion
    this.itemName = this.$table.data('item-name');
```

Reply:
Will do